K2Bridge.Tests.End2End/PrometheusTest.cs

```diff
@@ -60,23 +60,25 @@ public async Task PrometheusTelemetry_WhenQueryParsed_ThenExposeQueryMetrics()
                 responseData.Contains(
                 $"# HELP {metrics.AdxQueryDurationMetric.Name} {metrics.AdxQueryDurationMetric.Help}\n"
                 + $"# TYPE {metrics.AdxQueryDurationMetric.Name} histogram\n",
-                Ordinal), responseData);
+                Ordinal),
```

Comment:
This indentation feels wrong, why aren't the inner rows indented more?

Reply:
[SA1137](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1137.md) insists that line 63 should be the same as 61. 62 can be be pushed by a tab.

Reply:
When I format it, it turns into this (and that's how I think it should be):
```c#
            Assert.True(
                responseData.Contains(
                    "# HELP exceptions Exceptions logged\n"
                    + "# TYPE exceptions counter\n"
                    + "exceptions ",
                    Ordinal)
                );
```

I think we can control it with https://www.jetbrains.com/help/resharper/EditorConfig_CSHARP_CSharpIndentStylePageSchema.html#Parenthesis

`csharp_indent_method_decl_pars` and `csharp_indent_invocation_pars`